client_cli/pytests/test/roles/test_register_roles.py

```diff
@@ -0,0 +1,30 @@
+import allure
+import pytest
+
+@pytest.fixture(scope="function", autouse=True)
+def story_account_registers_roles():
+    allure.dynamic.story('Account registers a role')
+
+
+@allure.label('sdk_test_id', 'register_role')
+@pytest.mark.xfail(reason="TO DO")
+def test_register_role(
+        GIVEN_fake_name):
+    assert 0
+
+
+
+@allure.label('sdk_test_id', 'attach_permissions_to_role')
+@pytest.mark.xfail(reason="TO DO")
+def test_attach_permissions_to_role(
+        GIVEN_existence_asset_definition_with_store_value_type):
+    assert 0
+
+
+@allure.label('sdk_test_id', 'grant_role_to_account')
+@pytest.mark.xfail(reason="TO DO")
+def test_grant_role_to_account(
+        GIVEN_currently_authorized_account,
+        GIVEN_new_one_existence_account,
+        GIVEN_existence_asset_definition_with_store_value_type):
+    assert 0
```

Comment:
Missing newline. 

Omitting newlines at the end of files is fine for text, and even Rust source code, I've not complained about it earlier.

**However** It can lead to major breakages for interpreters other than CPython when crossing the Unix/Windows text encoding boundary. I suggest you add

Reply:
done